### PR TITLE
Fix link to conan repository

### PR DESCRIPTION
**Pull request by markaren**
Old URL was https://osp.jfrog.io/artifactory/api/conan/local
The correct one is ~https://osp.jfrog.io/artifactory/api/conan/conan-public~ https://osp.jfrog.io/artifactory/api/conan/conan-local

### CODE REVIEW

**Comment by markaren** [comment]
Or should it be conan-local ? Anyhow, the old link was dead.

File: README.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ following these steps:
 * Install [Conan] >= 1.7
 * Add the OSP Conan repository as a remote: 
 
-       conan remote add osp https://osp.jfrog.io/artifactory/api/conan/local
+       conan remote add osp https://osp.jfrog.io/artifactory/api/conan/conan-local
 * Package revisions must be enabled. See [How to activate the revisions].
```
Comment (hplatou) [inline README.md:28]: This actually also removes the requirement to enable package revisions.

**Review by hplatou — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (markaren) [reply README.md:28]: So remove the part mentioning package revisions in the README?

Comment (hplatou) [reply README.md:28]: Yes 👍

**Review by hplatou — APPROVED** [review]
👍

**Pull request merged**